cli/interaction.ss

```diff
@@ -257,31 +262,72 @@
              (option 'timeout-in-blocks "-T" "--timeout-in-blocks" default: #f
                      help: "number of blocks after which to time out")
              (option 'handshake "-H" "--handshake" default: #f
-                     help: "command to use to transfer handshakes")]
+                     help: "command to use to transfer handshakes")
+             ;; TODO: Abstract into Enum - See gerbil-poo
+             ;; enum off-chain-channel = 'stdio | 'libp2p
+             (option 'off-chain-channel-selection "-C" "--off-chain-channel" default: 'stdio
+                     help: "command to specify off-chain-channel")
+             (option 'host-address "-O" "--host-address" default: #f
+                     help: "host-address (only required if using libp2p as off-chain-channel)")
+             ;; FIXME: This should be stored and extracted from `dest-address`.
+             ;; It is supplied here as a temporary workaround.
+             (option 'dest-address "-D" "--dest-address" default: #f
+                     help: "dest-address (only required if using libp2p as off-chain-channel)")
+             (flag 'wait-for-agreement "-W" "--wait-for-agreement"
+                   help: "wait for agreement via off-chain-channel")]
             [(lambda (opt) (hash-remove! opt 'test))]
             [options/glow-path options/contacts
              options/evm-network options/database options/test options/backtrace]))
+
+  ;; NOTE: This also populates the runtime nickname-address mapping table.
+  (def contacts (load-contacts contacts-file))
+  ;; TODO: Validate whether you possess the necessary keys to assume this identity.
+  ;; TODO: Remove request for identity at stdio site.
```

Comment:
Avoiding this for now, since it will involve a rewrite of all integration tests, to send
identity earlier via stdin.
Have tracked this in the parent issue.